lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
+var inflect = require('i')();
+var _ = require('lodash');
+var EventSource = function() {
+};
```

Comment:
The Streaming change events section builds on SSE and should be compliant. 

The Request change events section describes getting the events using plain HTTP rest